test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/SigningUtilityTests.cs

```diff
@@ -68,6 +68,35 @@ public void Verify_WithValidInput_DoesNotThrow()
             }
         }
 
+        [CIOnlyFact]
+        public async Task SignAsync_WithSignedPackage_AddsRepositoryCountersignatureAsync()
```

Comment:
I recommend clarifying these new tests by adding a little more information to the test name.
```
SignAsync_WhenRepositoryCountersigningPrimarySignature_Succeeds
SignAsync_WhenRepositoryCountersigningRepositorySignedPackage_Throws
SignAsync_WhenRepositoryCountersigningRepositoryCountersignedPackage_Throws
```